server/persist/availabilityTest.test.js

```diff
@@ -0,0 +1,11 @@
+const getAvailability = require('../routes/getAvailability');
+
+
+test ('get all teachers', async function () {
+
+    let date = new Date ("2022-02-15");
+    const teachers = await getAvailability.getAvailablePeriods(date);
+    console.log(teachers);
```

Comment:
This test should check the expected value of `teachers` but I can add this to my initialization script pr

Reply:
sorry, i meant to delete this. I just used this to check if I was getting the correct result as I was working through this ticket.